snuba/datasets/errors_replacer.py

```diff
@@ -168,25 +168,32 @@ def set_project_exclude_groups(
     project_id: int,
     group_ids: Sequence[int],
     state_name: Optional[ReplacerState],
+    #  replacement type is just for metrics, not necessary for functionality
     replacement_type: ReplacementType,
 ) -> None:
     """
+    This method is called when a replacement comes in. For a specific project, record
+    the group ids which were deleted as a result of this replacement
+
```

Comment:
ok, this time, though, going on, I would discourage you from mixing cosmetics that are unrelated with your PR with the logic of your PR.
That makes reviewing harder because it means spending time to understand why a given change is there.

Reply:
Then in such case there should be first a PR with the refactoring and then a PR with the logic change.